modules/nw-anp-np-reference.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/logging-network-security.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-anp-differences-networkpolicy_{context}"]
+= Key differences between AdminNetworkPolicy and NetworkPolicy custom resources
+
+[cols="1,1,1"]
+|===
+|Policy elements | AdminNetworkPolicy | NetworkPolicy
+
+|Applicable user
+|Cluster administrator or equivalent
+|Developers within namespaces
```

Comment:
A
```suggestion
|Developers within namespaces
```
Tenant owners, namespace owners, application developers either of this works.